docs/SUBWORKFLOWS.md

```diff
@@ -0,0 +1,139 @@
+# Adding a new subworkflow to nf-scil
+
+## Generate the template
+
+First verify you are located at the root of this repository (not in `subworkflows`), then run the following interactive command :
+
+```
+nf-core subwworkflows create
+```
+
+That will ask you to give a name of your subworflow then the author name (github username).
+Alternatively, you can use the following command to supply directly those informations :
+
+```
+nf-core subworkflows create name --author
+```
+
+## Generate the template
+
+### Editing `./subworkflows/nf-scil/<name_of_your_workflow>/main.nf` :
+
+We don’t have the choice to generate an empty template when we generate a subworflow, so the template is based on nf-core.
+
+- remove the different comment lines.
+- include your modules into your subworkflows.
+  - remove the modules `{ SAMTOOLS_SORT}` and `{ SAMTOOLS_INDEX }` then includes yours with the good pathway:
```

Comment:
You can remove the first bullet and dedent the second. The second is a good enough description